eve/workers/end2end/scripts/run-e2e-test.sh

```diff
@@ -28,6 +29,13 @@ opts="
     --env=ZENKO_SESSION_TOKEN=${ZENKO_SESSION_TOKEN}
     --env=TOKEN=${TOKEN}
     --env=STAGE=${STAGE}
+    --env=OIDC_FULLNAME=${OIDC_FULLNAME}
+    --env=OIDC_USERNAME=${OIDC_USERNAME}
+    --env=OIDC_PASSWORD=${OIDC_PASSWORD}
+    --env=OIDC_ENDPOINT=${OIDC_ENDPOINT}
+    --env=OIDC_CLIENT_ID=${OIDC_CLIENT_ID}
+    --env=OIDC_REALM=${OIDC_REALM}
+    --env=UI_ENDPOINT=${UI_ENDPOINT}
```

Comment:
the `CYPRESS_`  envvars can be set here instead?

Reply:
indeed!